.env.example

```diff
@@ -39,3 +39,12 @@ PUSHER_APP_CLUSTER=mt1
 
 MIX_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
 MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
+
+# (currently, eloquent or ldap, defaults to eloquent if not defined)
+# CDASH_AUTHENTICATION_PROVIDER_DRIVER=eloquent
```

Comment:
To enable LDAP authentication set this to 'ldap', then configure the following env vars.